src/components/custom-antd/Button/Button.tsx

```diff
@@ -0,0 +1,7 @@
+import {Button as AntdButton, ButtonProps as AntdButtonProps} from 'antd';
+
+const Button: React.FC<AntdButtonProps> = props => {
+  return <AntdButton {...props} />;
```

Comment:
Should this be an entire component as it is not styled any different than the imported Button from antd?

Reply:
We will apply other styles to this component in future. And also it's a good practice to have antd components locally. It will help in case of restyling.